source/index.ts

```diff
@@ -52,7 +53,34 @@ const defaults: Partial<Defaults> = {
 		dnsCache: false,
 		useElectronNet: false,
 		responseType: 'text',
-		resolveBodyOnly: false
+		resolveBodyOnly: false,
+		pagination: {
+			parse: response => {
+				return JSON.parse(response.body);
+			},
+			paginate: response => {
```

Comment:
@sindresorhus I don't think we should implement a `paginate` function by default. Some people may want to go forwards, some will choose backwards, some will want to filter the elements etc.

Should `filter` be another property?

Reply:
> I don't think we should implement a paginate function by default. Some people may want to go forwards, some will choose backwards, some will want to filter the elements etc.

I still think it's nice to have a good default. Just getting all the pages or a set amount of items is the 90% use-case.

> Should filter be another property?

Yes